Cargo.toml

```diff
@@ -3,3 +3,6 @@ members = [
     "libsignal-service",
     "libsignal-service-actix",
 ]
+
+[patch.crates-io]
```

Comment:
Instead of patching crates.io, I guess you wanted to patch "https://github.com/Michael-F-Bryan/libsignal-protocol-rs"

Reply:
I just fixed it, thanks.